File: rwir/builtin/builtin.go
// Package builtin 提供 VM 内建算子求值引擎。
//
// 每个 op 文件通过 init() 自行注册 opcode 到全局容器：
//
//	func init() {
//	    Register("opcode", "sig", implementation)
//	}
package builtin

File: rwir/builtin/builtin.rs
//! Builtin dispatch table — identical to op/builtin/builtin.go and op/builtin/builtin.h.
pub fn register(_opcode: &str) { todo!("register") }
pub fn lookup(_opcode: &str) -> bool { false }
pub fn dispatch(_opcode: &str) -> String { todo!("dispatch") }
